src/components/userInfo/UserInfo.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
We probably won't need this component by the end so we should delete it before the code freeze